### DISCUSSION TITLE
How to run unit tests on `net6.0-maccatalyst` target?

### DISCUSSION THREAD

**Discussion by MartyIX** [nunit/nunit#4251] [Q&A] [answered]
Hi,

I have tried to run `dotnet test -f net6.0-maccatalyst -c Debug` on a mac machine (see https://github.com/MartyIX/libsodium-core/pull/2/checks) and it fails with:

```
  dotnet test -f net6.0-maccatalyst -c Debug
  shell: /bin/bash -e {0}
  env:
    DOTNET_ROOT: /Users/runner/.dotnet
  Determining projects to restore...
  All projects are up-to-date for restore.
/Users/runner/.dotnet/packs/Microsoft.MacCatalyst.Sdk/15.4.471/tools/msbuild/iOS/Xamarin.Shared.targets(578,3): error : A bundle identifier is required. Either add an 'ApplicationId' property in the project file, or add a 'CFBundleIdentifier' entry in the project's Info.plist file. [/Users/runner/work/libsodium-core/libsodium-core/test/Sodium.Tests/Sodium.Tests.csproj]
Error: Process completed with exit code 1.
```

Now I wonder whether it fails for some good reason or whether it is just unsupported at the moment. 

Could anyone please shed some light on this?

Best regards!

**Comment by OsirisTerje** [comment]
@MartyIX  I see no one could help out here.  But, did you figure this out ?

**Reply by MartyIX** [reply]
Hey, it's been a while so I don't really remember. I would try to add the following line:

https://github.com/dotnet/maui/blob/05b741dba520e6eb71faea4476ad0fb8a460af7f/src/Essentials/samples/Samples/Essentials.Sample.csproj#L17

but I don't remember if I could make it work. I think I didn't.

I have asked the same question in a xUnit's discussion https://github.com/xunit/xunit/issues/2314#issuecomment-1309894031 and it's not supported there. It just shows that `net6.0-maccatalyst` is a relatively new TFM.

**Reply by OsirisTerje** [reply, marked as answer]
Ok, understand.  I don't think there is anything we can do with this.  Suggest you follow this up with the dotnet team or perhaps even better the dotnet test team.
